package.json

```diff
@@ -496,122 +496,122 @@
             },
             {
                 "command": "calva.toggleCLJCSession",
-                "title": "Toggle the REPL connection (clj or cljs) used for CLJC files",
+                "title": "Toggle the REPL Connection (clj or cljs) used for CLJC Files",
                 "category": "Calva"
             },
             {
                 "command": "calva.selectCurrentForm",
-                "title": "Select current form",
+                "title": "Select Current Form",
                 "category": "Calva"
             },
             {
                 "command": "calva.clearInlineResults",
-                "title": "Clear all inline display of evaluation results",
+                "title": "Clear Inline Evaluation Results",
                 "category": "Calva"
             },
             {
                 "command": "calva.evaluateSelection",
-                "title": "Evaluate current form/selection inline and print to output",
+                "title": "Evaluate Current Form Inline",
                 "category": "Calva"
             },
             {
                 "command": "calva.evaluateCurrentTopLevelForm",
-                "title": "Evaluate current top level form inline and print to output (aka defun)",
+                "title": "Evaluate Top Level Form Inline",
                 "category": "Calva"
             },
             {
                 "command": "calva.evaluateSelectionPrettyPrint",
-                "title": "Evaluate current form/selection, and pretty print to output",
+                "title": "Evaluate Current Form Inline (pretty print)",
```

Comment:
These commands are gone. Or should be, with #344 .

Reply:
They are gone. *gg*